frontend/components/service/serviceCardRow.js

```diff
@@ -31,18 +31,20 @@ const CardContentNoPadding = styled(CardContent)(`
 `);
 
 const ConfirmDeleteDialog = (props) => {
-    const { open, setOpen, item, step, setStep, deleteService } = props;
+    const { open, setOpen, item, deleteService } = props;
     const handleClose = () => {
-        setStep(0);
+
         setOpen(false);
     };
     const handleConfirm = () => {
-        if (step === 0 && !item.hasOwnProperty('services')) {
-            setStep(1);
-        } else {
-            deleteService(item);
-            handleClose();
-        }
+        deleteService(item);
+        handleClose();
+        // if (step === 0 && !item.hasOwnProperty('services')) {
```

Comment:
We can remove the commented code, and retrieve later via version control if needed.